themes/vrctheme/404.php

```diff
@@ -14,9 +14,9 @@
 
 			<div class="page-wrapper">
 				<div class="page-content">
-					<h2><?php _e( 'This is somewhat embarrassing, isn’t it?'); ?></h2>
+					<h3><?php _e( 'It looks like nothing was found at this location. Maybe try a search?'); ?></h3>
                     <p><?php _e ( 'Could you please contact us below and notify us the broken page or link?' ); ?></p>
```

Comment:
Edit this paragraph for grammar.

Double check if you need "" or ''. Both are used, and we need to be consistent. If it doesn't matter, include a comment so that you can explain why it is okay either way if we get asked about it in our senior project presentation.